### PR TITLE
docs: fix pkg urls in quick-start instructions

### PR DESCRIPTION
**Pull request by marquiz**

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -20,15 +20,16 @@ First, install and setup cri-resource-manager.
 CRIRM_VERSION=`curl -s "https://api.github.com/repos/intel/cri-resource-manager/releases/latest" | \
                jq .tag_name | tr -d '"v'`
 source /etc/os-release
-sudo rpm -Uvh https://github.com/intel/cri-resource-manager/releases/download/v${CRIRM_VERSION}/cri-resource-manager-${CRIRM_VERSION}-0.x86_64.${ID}-${VERSION_ID}.rpm
+[ "$ID" = "sles" -o "$ID" = "opensuse-leap" ] && export ID=suse
+sudo rpm -Uvh https://github.com/intel/cri-resource-manager/releases/download/v${CRIRM_VERSION}/cri-resource-manager-${CRIRM_VERSION}-0.${ID}-${VERSION_ID}.x86_64.rpm
```
Comment (askervin) [inline docs/quick-start.md:24]: - Works on opensuse-15.2
- Fails on Fedora-33:
```
# sudo rpm -Uvh https://github.com/intel/cri-resource-manager/releases/download/v${CRIRM_VERSION}/cri-resource-manager-${CRIRM_VERSION}-0.${ID}-${VERSION_ID}.x86_64.rpm
Retrieving https://github.com/intel/cri-resource-manager/releases/download/v0.5.0/cri-resource-manager-0.5.0-0.fedora-33.x86_64.rpm
curl: (22) The requested URL returned error: 404
error: skipping https://github.com/intel/cri-resource-manager/releases/download/v0.5.0/cri-resource-manager-0.5.0-0.fedora-33.x86_64.rpm - transfer failed
```
  would work on Fedora 33 if `s/-${VERSION_ID}//'. Maybe we're missing "-33" in uploaded filenames. Or is it wanted?

File: docs/quick-start.md [hunk 2]
```diff
@@ -20,15 +20,16 @@ First, install and setup cri-resource-manager.
...
 ```
 
 #### Ubuntu\* and Debian\*
 ```
 CRIRM_VERSION=`curl -s "https://api.github.com/repos/intel/cri-resource-manager/releases/latest" | \
                jq .tag_name | tr -d '"v'`
 source /etc/os-release
-pkg=cri-resource-manager_${CRIRM_VERSION}_amd64.${ID}-${VERSION_ID}.deb; curl -LO https://github.com/intel/cri-resource-manager/releases/download/v${CRIRM_VERSION}/${pkg}; sudo dpkg -i ${pkg}; rm ${pkg}
+pkg=cri-resource-manager_${CRIRM_VERSION}_${ID}-${VERSION_ID}_amd64.deb; curl -LO https://github.com/intel/cri-resource-manager/releases/download/v${CRIRM_VERSION}/${pkg}; sudo dpkg -i ${pkg}; rm ${pkg}
```
Comment (askervin) [inline docs/quick-start.md:32]: - Works on Ubuntu 20.04
- Works on Ubuntu 18.04
- Works on Debian 10
- Fails on Debian-sid. VERSION_ID is not set. Would work if VERSION_ID=sid. However, the only variable containing "sid" in /etc/os-release is PRETTY_NAME...
```
debian@e2e-debian-sid-containerd:~$ cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux bullseye/sid"
NAME="Debian GNU/Linux"
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

Comment (klihub) [reply docs/quick-start.md:32]: @marquiz Any updates planned to this PR to address debian-sid failure reported by @askervin, or do you consider this good to go ?

Comment (marquiz) [reply docs/quick-start.md:32]: I'm not planning to invest more in this PR. Any further improvements left as a future exercise 😄

**Review by klihub — APPROVED** [review]

**Pull request merged**